mitodl/mit-open
# allow filtering by readable id in the api

### What are the relevant tickets?

Closes #604 

### Description (What does it do?)
Adds "readable_id" as a filterable field in learning_resource apis


### How can this be tested?
1. Checkout this branch; make sure you have learning resources locally.
2. visit a learning resources api such as /api/v1/courses/ and grab a readable_id value
3. url encode this value ( course-v1:xPRO+external1+R1 -> course-v1%3AxPRO%2Bexternal1%2BR1 ) 
4. use it as a filter - /api/v1/courses/?readable_id=course-v1%3AxPRO%2Bexternal1%2BR1


### Checklist:
- [ ] tested this filter on all resource types (podcasts, courses, etc)
